protostar/cheatable_starknet/cheatables/cheatable_syscall_handler.py

```diff
@@ -36,7 +38,7 @@ def __init__(self, message: str):
         super().__init__(message)
 
 
-class CheatableSysCallHandler(BusinessLogicSysCallHandler):
+class CheatableSysCallHandler(DeprecatedBlSyscallHandler):
```

Comment:
`segments` is no longer needed, it's injected higher up in the original OS implementation